public/static/docs/command-reference/config.md

```diff
@@ -152,6 +152,13 @@ for more details.) This section contains the following options:
   > These warnings are automatically turned off when `cache.type` is manually
   > set.
 
+- `cache.shared` - permissions for newly created or downloaded cache files and
+  directories. The default permissions are `0o664`(rw-r--r--) for files and
+  `0o755`(rwxr-xr-x) for directories. The only accepted value right now is
```

Comment:
Is presenting permissions in such way clear enough? I'm not sure if it is too hard to understand, maybe we are better describing it with words to make it as friendly as possible?